kafka-admin/src/main/resources/application.properties

```diff
@@ -36,6 +36,11 @@ mp.openapi.scan.disable=false
 mp.openapi.filter=org.bf2.admin.kafka.admin.handlers.OASModelFilter
 quarkus.smallrye-openapi.info-version=${quarkus.application.version:0.0.1-SNAPSHOT}
 
+quarkus.swagger-ui.enable=true
+quarkus.swagger-ui.always-include=true
+quarkus.swagger-ui.title=Kafka Instance Admin
+quarkus.swagger-ui.theme=monokaig
```

Comment:
Should this be `monokai` (no `g`) ? I get an error with this value.

Reply:
```suggestion
quarkus.swagger-ui.theme=monokai
```

Reply:
My bad. Made typo